analyzers/src/SonarAnalyzer.CSharp/Rules/UnnecessaryUsings.cs

```diff
@@ -85,6 +85,12 @@ private static void CheckUnnecessaryUsings(SyntaxNodeAnalysisContext context, IE
         {
             foreach (var usingDirective in usingDirectives)
             {
+                // This will create some FNs but will kill noise from FPs.
+                // For more info see issue: https://github.com/SonarSource/sonar-dotnet/issues/5946
+                if (usingDirective.GetFirstToken().IsKind(SyntaxKind.GlobalKeyword))
```

Comment:
This is more of a question than a remark, if it exists does the global keyword always have to be the first token in the row?

Reply:
For using directives if there's global keyword should always be first.
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-10.0/globalusingdirective